### PR TITLE
Welcome sidebar settings

### LINKED ISSUE #160: Input from Linux Game Cast: LWDW 352
See [LWDW 352](https://linuxgamecast.com/2022/11/lwdw-351-fedora-37-and-davinci-resolve-18-1/#).

- [ ] The sidebar should be easier to disable (should this be an option in the startup options?)

P.S. If you are the original author, we are always open to feedback

### PR DESCRIPTION
**Pull request by PressJump**
Fixes #160 adding sidebar preference settings to setup and Depends on https://github.com/pulse-browser/browser/pull/159

image

### CODE REVIEW

File: src/browser/locales/en-US/browser/welcome.ftl [hunk 1]
```diff
@@ -16,7 +16,13 @@ welcome-dialog-theme = Choose a theme
 welcome-dialog-theme-action = Continue
 
 welcome-dialog-search = Chose a search engine
-welcome-dialog-search-action = Done
+welcome-dialog-search-action = Next
+
+welcome-dialog-sidebar = Choose your Sidebar Settigns
```
Comment (splatboydev) [inline src/browser/locales/en-US/browser/welcome.ftl]: Settings

**Comment by trickypr** [comment]
#159 needs significant changes

**Comment by PressJump** [comment]
I will be updating this PR with changes made to sidebar-active-tabs that was merged to branch alpha. Leaving it draft until then.

**Comment by PressJump** [comment]
@trickypr Ready to review and up to date with alpha.

**Review by trickypr — APPROVED** [review]

**Pull request merged**